components/badges/badges.css

```diff
@@ -0,0 +1,122 @@
+@import "/reset.css";
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 80px;
+    justify-content: center;
+    gap: 60px;
+}
+.badges{
+    display: flex;
+    flex-direction: c;
+    align-items: center;
+    justify-content: center;
+    gap: 25px;
+}
+.text__badge{
+    position: relative;
+    width: 160px;
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
have a utility class, it will save your time

Reply:
done